fluent.syntax/tests/syntax/test_literal.py

```diff
@@ -0,0 +1,145 @@
+# coding: utf-8
```

Comment:
Note on this test, it's heavily inspired by the js tests, but I took a shortcut in the fixtures in a place or two.

Also, the `U{6}` tests don't work on standard python2, as it doesn't have wide unichar support, so I made that depend on sys.